omero-installation/omero_installation_centos6_sclpython27.md

```diff
@@ -0,0 +1,209 @@
+### OMERO.server installation on CentOS 6 with scl Python 2.7 and Ice 3.5
+
+This is an example walkthrough for installing OMERO 5.2.x on CentOS 6 with scl Python 2.7, using a dedicated system user.
+These instructions assume your Linux distribution is configured with a UTF-8 locale.
+
+#### Name: OMERO team
+
+#### Institute: University of Dundee
+
+#### Contact: OME External Developer List <ome-devel@lists.openmicroscopy.org.uk>
+
+#### Context:
+
+For convenience in this walkthrough the main OMERO configuration options have been defined as environment variables. When following this walkthrough you should use your own values:
+ 
+  	OMERO_DB_USER=db_user
+	OMERO_DB_PASS=db_password
+	OMERO_DB_NAME=omero_database
+	OMERO_ROOT_PASS=omero_root_password
+	OMERO_DATA_DIR=/OMERO
+
+	OMERO_WEB_PORT=80
+
+	export OMERO_DB_USER OMERO_DB_PASS OMERO_DB_NAME OMERO_ROOT_PASS OMERO_DATA_DIR OMERO_WEB_PORT
+
+	export PGPASSWORD="$OMERO_DB_PASS"
+
+#### System Specifications:
+
+CentOS 6 with scl Python 2.7, Java 1.8, Ice 3.5 and PostgreSQL 9.4. Nginx 1.8 for OMERO.web
+
+#### Installation:
+
+The following steps describe how to install an OMERO.server and to deploy OMERO.web
+using Nginx.
+We assume that common packages like ``wget``, ``unzip``, ``tar`` are already installed.
+
+##### Step 1:
+
+As **root**, Install:
+
+Various dependencies:
+	
+	yum -y install epel-release
+	yum -y install centos-release-SCL
+
+	yum -y install \
+	python27 \
+	python27-virtualenv \
+	python27-numpy \
+	libjpeg-devel \
+	libpng-devel \
+	libtiff-devel \
+	zlib-devel \
+	hdf5-devel \
+	freetype-devel \
+	expat-devel
+
+Java 1.8:
+
+	yum -y install java-1.8.0-openjdk
+
+Others packages required for Ice 3.5:
+
+	yum -y groupinstall "Development Tools"
+
+pip and few requirements for OMERO:
+
+	set +u
+	source /opt/rh/python27/enable
+	set -u
+	easy_install pip
+
+	export PYTHONWARNINGS="ignore:Unverified HTTPS request"
+
+	pip install tables
+	pip install matplotlib
+	pip install "Pillow<3.0"
+
+Ice 3.5:
+
+	curl -o /etc/yum.repos.d/zeroc-ice-el6.repo \
+	http://download.zeroc.com/Ice/3.5/el6/zeroc-ice-el6.repo
+
+	yum -y install db53 db53-utils mcpp
+
+	mkdir /tmp/ice-download
+	cd /tmp/ice-download
+
+	wget http://downloads.openmicroscopy.org/ice/experimental/Ice-3.5.1-b1-centos6-sclpy27-x86_64.tar.gz
+
+	tar -zxvf /tmp/ice-download/Ice-3.5.1-b1-centos6-sclpy27-x86_64.tar.gz
+
+	# Install under /opt
+	mv Ice-3.5.1-b1-centos6-sclpy27-x86_64 /opt/Ice-3.5.1
+
+	# make path to Ice globally accessible
+	# if globally set, there is no need to export LD_LIBRARY_PATH
+	echo /opt/Ice-3.5.1/lib64 > /etc/ld.so.conf.d/ice-x86_64.conf
+	ldconfig
+
+
+Postgres 9.4 and reconfigure to allow TCP connections:
+
+	yum -y install postgresql94-server postgresql94
+
+	service postgresql-9.4 initdb
+	sed -i.bak -re 's/^(host.*)ident/\1md5/' /var/lib/pgsql/9.4/data/pg_hba.conf
+	chkconfig postgresql-9.4 on
+	service postgresql-9.4 start
+
+##### Step 2:
```

Comment:
I cannot see the difference in the number of characters here, but evidently this header is considered to be formatted a bit differently than, e.g. Step 3; (which is blue and not shown as italics)